test/unit/SourceLocator/SourceStubber/ReflectionSourceStubberTest.php

```diff
@@ -295,11 +295,16 @@ private function assertSameParameterAttributes(
         CoreReflectionParameter $original,
         ReflectionParameter $stubbed,
     ): void {
-        $parameterName = $original->getDeclaringClass()->getName()
-            . '#' . $originalMethod->getName()
-            . '.' . $original->getName();
+        $methodName    = $original->getDeclaringClass()->getName() . '#' . $originalMethod->getName();
+        $parameterName = $methodName . '.' . $original->getName();
 
         self::assertSame($original->getName(), $stubbed->getName(), $parameterName);
+
+        // PHP sometimes reports default value available but it throws error when trying to get the value
```

Comment:
This comment seems to be out of sync with the code: I guess we have more default values declared in stubs than in PHP?

Reply:
I've to check it again. I'm not able to understand my yesterday's code...

Reply:
It looks I fixed something else in the night... Fixed the right place now :)